arangod/Agency/AsyncAgencyComm.cpp

```diff
@@ -282,6 +282,18 @@ arangodb::AsyncAgencyComm::FutureResult agencyAsyncSend(AsyncAgencyCommManager&
 
 namespace arangodb {
 
+AsyncAgencyComm::FutureResult AsyncAgencyComm::sendWithFailover(
+    arangodb::fuerte::RestVerb method, std::string const& url, network::Timeout timeout,
+    RequestType type, uint64_t index) const {
+  std::vector<ClientId> clientIds;
+  VPackBuffer<uint8_t> body;
+  {
+    VPackBuilder builder(body);
+    builder.add(VPackValue(index));
+  }
+  return sendWithFailover(method, url + "?index=" + std::to_string(index), timeout, type, clientIds, std::move(body));
```

Comment:
so we need `index` in the body as well as a query parameter? O_o

Reply:
👍 +1: